.travis.yml

```diff
@@ -4,7 +4,7 @@ cache:
     - "$HOME/.m2"
 language: java
 script:
-  - mvn -B -T2 -fae -Djava.awt.headless=true -Dall clean install
+  - mvn -B -fae -Djava.awt.headless=true -Dall clean install
```

Comment:
Hi
Sorry, I may have included some stuff in my pull request I shouldn't.
I'm still newby with github, and I don't know why my pull request included commits from andrea and others...
And yes, I did a rebase and get error messages from this rebase.

Michaël

________________________________
De : Jody Garnett [notifications@github.com]
Date d'envoi : jeudi 27 août 2015 23:13
À : geotools/geotools
Cc : Michael Michaud
Objet : Re: [geotools] Improvements of LineLabel patch "letter level conflict management" according to Andrea's advices : suppression of dead code, refactoring of glyph processing. (#948)


In .travis.yml<https://github.com/geotools/geotools/pull/948#discussion_r38146843>:

> @@ -4,7 +4,7 @@ cache:
>      - "$HOME/.m2"
>  language: java
>  script:
> -  - mvn -B -T2 -fae -Djava.awt.headless=true -Dall clean install
> +  - mvn -B -fae -Djava.awt.headless=true -Dall clean install


Interesting changing build settings as part of this pull request - did something happen during a rebase?

—
Reply to this email directly or view it on GitHub<https://github.com/geotools/geotools/pull/948/files#r38146843>.